cloud/keystone_test_utils.go

```diff
@@ -186,3 +186,16 @@ func getV3TenantsResponse() interface{} {
 		},
 	}
 }
+
+func getV3DomainsResponse() interface{} {
+	return map[string]interface{}{
+		"domains": []interface{}{
+			map[string]interface{}{
+				"id":          "Default",
+				"name":        "default",
```

Comment:
This is test data so it doesn't really matter, but in reality it's the other way around: id is "default" and name is "Default".

Reply:
done